docs/src/docs/pages/progress-bar-page/index.tsx

```diff
@@ -37,18 +37,18 @@ const ProgressBarPage = () => {
       <Stack spacing={ 4 } maxW="container.md">
         <Stack mb={ 4 } spacing={ 4 }>
           <Text>Mediatool uses a custom progress bar component</Text>
-          <ProgressBar loaded={ 0.5 } />
-          <Code w="max-content">{ '<ProgressBar loaded={ 0.5 } />' }</Code>
+          <LoadingBar loaded={ 0.5 } />
+          <Code w="max-content">{ '<LoadingBar loaded={ 0.5 } />' }</Code>
           <Text>
             It takes a <b>loaded</b> prop which is a decimal between 0 and 1 indicating
             the percentage of how much has loaded
           </Text>
-          <ProgressBar loaded={ progress } />
+          <LoadingBar loaded={ progress } />
           <Button onClick={ handleClick }>Start Loading</Button>
           <Code w="max-content" display="block" whiteSpace="pre">
             { `setInterval(() => setProgress((prev) => prev + Math.random() * 0.1), Math.random() * 300 + 200)
 
-<ProgressBar loaded={ progress }` }
+<LoadingBr loaded={ progress }` }
```

Comment:
typo, it should be `LoadingBar`. is ESLint not set up properly?

Reply:
Eslint